src/main/java/at/hannibal2/skyhanni/data/HotmData.kt

```diff
@@ -57,50 +56,50 @@ private fun calculatePeakOfTheMountainLoot(level: Int): Map<HotmReward, Double>
 enum class HotmData(
     val guiName: String,
     val maxLevel: Int,
-    val costFun: (Int) -> (Double?),
+    val costFun: (Int) -> (Int?),
     val rewardFun: (Int) -> (Map<HotmReward, Double>),
 ) {
 
     MINING_SPEED(
         "Mining Speed",
         50,
-        { currentLevel -> (currentLevel + 2.0).pow(3) },
+        { level -> (level + 1.0).pow(3).toInt() },
         { level -> mapOf(HotmReward.MINING_SPEED to level * 20.0) },
     ),
     MINING_FORTUNE(
         "Mining Fortune",
         50,
-        { currentLevel -> (currentLevel + 2.0).pow(3.05) },
+        { level -> (level + 1.0).pow(3.05).toInt() },
         { level -> mapOf(HotmReward.MINING_FORTUNE to level * 5.0) },
     ),
     QUICK_FORGE(
         "Quick Forge",
         20,
-        { currentLevel -> (currentLevel + 2.0).pow(4) },
-        { level -> mapOf(HotmReward.FORGE_TIME_DECREASE to 10.0 + (level * 0.5)) },
+        { level -> (level + 1.0).pow(4).toInt() },
+        { level -> mapOf(HotmReward.FORGE_TIME_DECREASE to if (level >= 20) 30.0 else 10.0 + (level * 0.5)) },
```

Comment:
are you sure that it does not work with blue egg?

Reply:
oh, the updated the wiki formula
```
Reward Formula: min(30, 10+(Level*0.5)+floor(Level/20)*20)
```

Reply:
Yeah, I'm pretty sure blue egg doesn't buff it further, but I can have someone confirm.

Reply:
Confirmed.
<img width="370" alt="image" src="https://github.com/hannibal002/SkyHanni/assets/148620615/6f39a4be-282e-45d2-8b68-1b2b82b51c81">